src/main/main.ts

```diff
@@ -103,6 +105,7 @@ const client = new NTPClient("time.google.com", 123, { timeout: 5000 });
 
 let remoteHeadless: RemoteHeadless;
 let useRemoteHeadless: boolean;
+let remoteNode: NodeInfo;
```

Comment:
How can I move Apollo initialization?

Reply:
Assuming we have `remoteNode` determined at the start of process and it doesn't change afterwards, we could make a quick handler like this in main process:

```ts
ipcMain.on("get remote node", event => event.returnValue = remoteNode);
```

and call them as sendSync like this: `ipcRenderer.sendSync("get remote node")`. Of course this has performance implications as well but since we are just returning same value and we wouldn't make a lot of calls to them I think it should be fine.